python/servo/build_commands.py

```diff
@@ -25,8 +24,14 @@
 from servo.command_base import CommandBase, cd
 
 
-def is_headless_build():
```

Comment:
We can't land this PR without it, and we can't change the CI without this PR. I'm saying we do this step by step :)

But closing the tree, updating CI, and landing this would work too.